soh/soh/GameMenuBar.cpp

```diff
@@ -617,6 +617,15 @@ namespace GameMenuBar {
                         ImGui::EndMenu();
                     }
 
+                    UIWidgets::Spacer(0);
+
+                    bool disabled = !CVarGetInteger("gSuperSonic", 0);
+                    const char* disabledTooltip = "This option is disabled because \"Super Sonic Mode\" is turned off";
```

Comment:
```suggestion
```

Reply:
Following up my comment below, keeping them as variables is preferable in my eyes, but I'd rename the variables to be unique to not potentially clash with others. So `sonicModeDisabled` and ``sonicModeDisabledTooltip` or whatever the name ends up being.

Reply:
Yeah, using vars is fine too, I just didn't want to propose new var names until the feature name was decided on